code/drasil-lang/lib/Drasil/Language/Idea.hs

```diff
@@ -1,6 +1,6 @@
 {-# LANGUAGE TemplateHaskell #-}
 -- | The lowest level of chunks in Drasil. It all starts with an identifier and a term.
-module Language.Drasil.Chunk.NamedIdea (
+module Drasil.Language.Idea (
```

Comment:
I don't see the point of making this change right now. It's just shuffling things around *before* we have a decent design.

Note that it may end up being a fine change. I don't know, either way, right now.